CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ which will stop push notifications registering (and potentially asking for permi
 
 ## Video
 - Xamarin Mac was upgraded to support 64-bit executables on macOS.
+- Added some JavaScript methods to `Video` to make it easier to control playback from JavaScript, as well as obtaining information about playback position and video duration.
```

Comment:
This is now in the wrong section of the changelog since we did 1.3. Would you mind rebasing, on master and moving this to the "unreleased"-section, @kristianhasselknippe?

Reply:
OK, so this description is going to need updating as well now. We no longer expose position to JS.